sklearn/model_selection/_split.py

```diff
@@ -1050,7 +1050,7 @@ class StratifiedShuffleSplit(BaseShuffleSplit):
     >>> y = np.array([0, 0, 1, 1])
     >>> sss = StratifiedShuffleSplit(n_iter=3, test_size=0.5, random_state=0)
     >>> sss.get_n_splits(X, y)
-    3
+    3n
```

Comment:
I fixed the typo and I'll take a closer look at why the docstring didn't seem to be run.